lib/langParser.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+const iso639 = require('iso-639-1')
+
+fs.readdir(`${__dirname}/../static/i18n`, function(err, items) {
+  var avaliableLanguages = {
```

Comment:
Typo - should be `availableLanguages`.

Also, use ES6's `const` instead of `var`.